kubernetes/.openapi-generator/COMMIT

```diff
@@ -0,0 +1,2 @@
+Requested Commit: v3.3.4
```

Comment:
`kubernete/.swagger-codegen` is deleted, openapi-generator uses `kubernetes/.openapi-generator` and also `.openapi-generator-ignore` instead of `.swagger-codegen-ignore` (renamed).